### ISSUE TITLE
Not finding freshly installed hdf5 1.12 on Windows 10

### ISSUE THREAD

**Issue by thomasaarholt** [aldanor/hdf5-rust#102]
Hiya,

I've just tried installing [HDF5 1.12](https://www.hdfgroup.org/downloads/hdf5) and restarting my Windows 10 computer.
I have not previously tried running hdf5-rust, so I don't know if this is something to do with the latest version, my Windows install, or hdf5-rust. My hdf5 install is located at `C:\Program Files\HDF_Group\HDF5\1.12.0`, (this folder contains the `bin`, `include`, etc. folders).

Running the example from the readme, I'm getting
```rust
Caused by:
  process didn't exit successfully: `C:\Users\thomasaar\Documents\GitHub\example\target\debug\build\hdf5-sys-c8a649b3d21048b6\build-script-build` (exit code: 101)
--- stdout
Searching for installed HDF5 (any version)...
Found no HDF5 installations.
```

**Comment by thomasaarholt** [comment]
Since the code tells me it might be relevant, here is a screenshot of the regedit properties of the uninstall reg for the hdf5 installation:
![image](https://user-images.githubusercontent.com/2721423/88545123-d3d68580-d01a-11ea-9f72-eb3176856bc2.png)

**Comment by magnusuMET** [comment]
Hi, this crate is not yet compatible with 1.12, see #94 and #95. If you try PR #95 you could set HDF5_DIR and the crate should compile with this version. You can add this to your `Cargo.toml`
```toml
[patch]
[patch.crates.io]
hdf5 = { git = "https://github.com/mulimoen/hdf5-rust.git", branch="feature/hdf5-1.12.0" }
```

Another option is building `HDF5` from source and linking statically. To do this you can add
```toml
[patch]
[patch.crates.io]
hdf5 = { git = "https://github.com/aldanor/hdf5-rust.git", features = ["hdf5-sys/static"] }
```

**Comment by thomasaarholt** [comment]
Excellent! I'm new to Rust, so this is a nice learning experience. I'll try this tomorrow!

**Comment by thomasaarholt** [comment]
The first suggestion worked beautifully! Just one little typo: `[patch.crates.io]` -> `[patch.crates-io]`.

**Comment by aldanor** [comment]
Should be fixed by #95
